src/quo2/foundations/resources.cljs

```diff
@@ -25,10 +37,18 @@
   (get tokens k))
 
 (def networks
-  {:ethereum (js/require "../resources/images/tokens/mainnet/ETH-network.png")
-   :optimism (js/require "../resources/images/tokens/mainnet/OP.png")
-   :arbitrum (js/require "../resources/images/tokens/mainnet/ARB.png")})
+  {:arbitrum (js/require "../resources/images/networks/Arbitrum.png")
+   :ethereum (js/require "../resources/images/networks/Ethereum.png")
+   :gnosis   (js/require "../resources/images/networks/Gnosis.png")
+   :hermez   (js/require "../resources/images/networks/Hermez.png")
+   :optimism (js/require "../resources/images/networks/Optimism.png")
+   :polygon  (js/require "../resources/images/networks/Polygon.png")
+   :scroll   (js/require "../resources/images/networks/Scroll.png")
+   :taiko    (js/require "../resources/images/networks/Taiko.png")
+   :unknown  (js/require "../resources/images/networks/Unknown.png")
+   :xdai     (js/require "../resources/images/networks/xDAI.png")
+   :zksync   (js/require "../resources/images/networks/zkSync.png")})
 
 (defn get-network
-  [n]
-  (get networks n))
+  [k]
+  (get networks k))
```

Comment:
Just a side comment, maybe too late:

the `networks` var is already a map, and we can use a map as a function to extract its keys, so this `get-network` function is useless, it's just a wrapper that adds nothing:

```clojure
(def m {:a 1 :b 2})

(m :a)
;; => 1

(get m :a)
;; => 1

(m :x)
;;=> nil

(get m :x)
;;=> nil
```

Reply:
@ulisesmac I agree!

If it's too late, or if we like the idiom of `get-image`, `get-dapp`, `get-foo`, we could try to make them more valuable like this (not sure if we'll ever vary those base-paths so maybe it's ok):

```clojure
(def networks
  {:arbitrum "Arbitrum.png"
   :ethereum "Ethereum.png"
   :gnosis   "Gnosis.png"
   :hermez   "Hermez.png"
   :optimism "Optimism.png"
   :polygon  "Polygon.png"
   :scroll   "Scroll.png"
   :taiko    "Taiko.png"
   :unknown  "Unknown.png"
   :xdai     "xDAI.png"
   :zksync   "zkSync.png"})

(defn get-network
  [k]
  (js/require (str "../resources/images/networks/" (networks k))))
  ```

Reply:
@erikseppanen The (potential) problem I see is the `js/require` call will be performed each time we want to use a resource instead of loading them in the app initialization and extract them from a map. :thinking:  Not sure if it's actually worse TBH.

To add value, maybe we could define a fallback option:

```clojure
(def networks
  {:ethereum              (js/require (str "..." ,,,))
   :not-found-placeholder (js/require (str "..." ,,,))}

(defn get-network [k]
 (get networks k (networks :not-found-placeholder))
```

Reply:
@ulisesmac good point re: app initialization, it's probably worse in that respect 👍

Reply:
@ulisesmac - IMO, the reason to use `get-network` is to add the fallback (not found) in one place, rather than refactoring every usage.